client/signup/main.jsx

```diff
@@ -553,9 +565,6 @@ class Signup extends React.Component {
 			return null;
 		}
 
-		const flow = flows.getFlow( this.props.flowName );
-		const showProgressIndicator = 'pressable-nux' === this.props.flowName ? false : true;
```

Comment:
`pressable-nux` is still an active flow. Did you mean to remove this?

Reply:
Nice catch. I _thought_ I'd removed it in #26532 but you're right. Should bring that back.